day20/C++/PartitionDay20.cpp

```diff
@@ -11,6 +11,7 @@ using namespace std;
 
 int main(){
 	int n;
+	cout << "Enter the number of values you want to have."<<endl;
 	cin >> n;
 	int a[n];
 	for(int i = 0; i < n; i++){
```

Comment:
You should also add a print statement to inform user that he needs to enter a value (integer) each time.

Reply:
Hey aadikamat,
Thanks for the suggestion. I have done the suggested changes.

Reply:
I don't see a print statement before line 18 though.